tesseract/tesseract_environment/src/core/environment.cpp

```diff
@@ -185,6 +185,19 @@ bool Environment::moveJoint(const std::string& joint_name, const std::string& pa
   return true;
 }
 
+bool Environment::changeJointOrigin(const std::string& joint_name, const Eigen::Isometry3d& new_origin)
+{
+  if (!scene_graph_->changeJointOrigin(joint_name, new_origin))
+    return false;
+
+  ++revision_;
+  commands_.push_back(std::make_shared<ChangeLinkOriginCommand>(joint_name, new_origin));
```

Comment:
Should this command be `ChangeJointOriginCommand`?

Reply:
Yes. I will change that.